test/check-dependencies-specs.js

```diff
@@ -0,0 +1,82 @@
+import { checkForDependencies, BOOTSTRAP_PATH } from '..';
```

Comment:
I'd rather start with moving WebDriverAgent class from xcutest driver to this package. It might we'll need to change the source a lot, which will create more work in unit tests update.

Reply:
This functionality that is being tested should remain the same, even if we move the entirety of the `WDA` stuff over from `appium-xcuitest-driver`. These tests are just copied from there anyway.